alerting/src/test/kotlin/org/opensearch/alerting/settings/AlertingSettingsIT.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.settings
+
+import org.opensearch.alerting.ALWAYS_RUN
+import org.opensearch.alerting.AlertingRestTestCase
+import org.opensearch.alerting.model.Trigger
+import org.opensearch.alerting.model.action.Action
+import org.opensearch.alerting.randomAction
+import org.opensearch.alerting.randomQueryLevelMonitor
+import org.opensearch.alerting.randomQueryLevelTrigger
+
+class AlertingSettingsIT : AlertingRestTestCase() {
+
+    fun `test updating setting of max actions per trigger with less actions than allowed actions`() {
+        val actions = createActions(4)
+        val triggers = createTriggers(1, actions)
+        createMonitor(
+            randomQueryLevelMonitor(
+                triggers = triggers
+            )
+        )
+
+        try {
+            client().updateSettings(AlertingSettings.TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS.key, 1)
+        } catch (e: Exception) {
+            assertTrue(e is IllegalArgumentException)
+        }
+    }
+
+    fun `test updating setting of max actions per trigger with more actions than allowed actions`() {
```

Comment:
Test name does not make sense. This is testing if the user is able successfully update the setting with a valid value. Name is something along the lines of: `test updating setting of max actions per trigger with an existing monitor successfully`.

Also do a getSettings call to verify the setting got updated